src/cmd-compress

```diff
@@ -12,10 +12,21 @@ import argparse
 sys.path.insert(0, os.path.dirname(os.path.abspath(__file__)))
 from cmdlib import run_verbose, write_json, sha256sum_file, rm_allow_noent
 
+# Hardcoded value for compression algorithm for now
+DEFAULT_COMPRESSION_ALGO = 'xz'
+
 parser = argparse.ArgumentParser()
 parser.add_argument("--build", help="Build ID")
+parser.add_argument("--compression-algorithm",
```

Comment:
Slight bikeshed: how about just `--algorithm`? Since the command is already called `compress` it should be clear enough I think.

Reply:
Sure, will be updating